extensions/wikia/Wall/Wall.setup.php

```diff
@@ -100,7 +100,7 @@
 // watchlist
 $wgHooks['ArticleCommentBeforeWatchlistAdd'][] = 'WallHooksHelper::onArticleCommentBeforeWatchlistAdd';
 // $wgHooks['WatchArticle'][] = 'WallHooksHelper::onWatchArticle';
-// $wgHooks['UnwatchArticle'][] = 'WallHooksHelper::onUnwatchArticle';
+$wgHooks['UnwatchArticle'][] = 'WallHooksHelper::onUnwatchArticle';
```

Comment:
These lines were [commented out in 2012](https://github.com/Wikia/app/commit/5d7f81c4057e17a2b225f55055660f9b14bd2147) by Tomach with the commit message "follow fix for Wall".  No ticket #, no other explanation.  Laffs.

The code at the other end of this limits itself to only unwatching things that look like threads or things that aren't the wall itself.